applications/FluidDynamicsApplication/python_scripts/response_function_output_process.py

```diff
@@ -0,0 +1,70 @@
+# Importing the Kratos Library
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+# other imports
+from KratosMultiphysics.time_based_ascii_file_writer_utility import TimeBasedAsciiFileWriterUtility
+
+def Factory(settings, Model):
+    if(type(settings) != Kratos.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return ResponseFunctionOutputProcess(Model, settings["Parameters"])
+
+class ResponseFunctionOutputProcess(Kratos.Process):
```

Comment:
This must derive from the new `OutputProcess`.

Reply:
done :)